### ISSUE TITLE
[tune] import errors and  unexpected keyword argument 'metric'

### ISSUE THREAD

**Issue by yushan111** [ray-project/ray#5075]
### System information
- **OS Platform and Distribution: Linux Ubuntu 16.04**:
- **Ray installed from `pip install ray`**:
- **Ray version: 0.7.1**:
- **Python version: 3.6.0**:
- **Exact command to reproduce: python tune_mnist_keras.py**:

### Describe the problem
1. when I run tune_mninst_keras.py, I got an error of "No module named 'ray.tune.integration'". I found there is no 'integration' in my python site-packages. I installed ray by pip.
2. Can you change the corresponding ray tune examples while calling AsyncHyperBandScheduler(), since you have modified the name of `reward_attr` and there is no `mode` in the class anymore.

**Comment by richardliaw** [comment]
This is the same error - https://github.com/ray-project/ray/issues/5026

**Comment by richardliaw** [comment]
This should be included in the newest ray `pip install -U ray`.
